lib/Map/Legend.js

```diff
@@ -115,36 +115,24 @@ var Legend = function(props) {
      */
     this.variableNameTop = defaultValue(props.variableNameTop, 17);
 
+    /**
+     * Gets or sets the CSS class that will be applied to the legend when it is displayed.
+     * This is used to ensure that the correct font is used when measuring text for word wrapping.
+     * @type {String}
+     */
+    this.cssClass = defaultValue(props.cssClass, 'legend__legend');
```

Comment:
This is the dodgiest bit of the pull request.  In order to do manual word wrapping, we need to know what font will be used.  The fundamental problem is that all the code that creates legends has no actual idea of what CSS will eventually be used to render them.  So putting `legend__legend` here is a pretty serious violation of the principle that the `Map` layer shouldn't know about the `Views` layer.  Ideas on how to fix this are welcome, otherwise I guess we'll just live with it.